src/callbacks/app.js

```diff
@@ -0,0 +1,39 @@
+const request = require('request')
+const weatherUrl = [
+    'https://api.openweathermap.org/data/2.5/onecall?lat=',
+    '&lon=',
+    '&appid=6bf6d4d79901e20e0f4c99afbe44ec8a'
+]
+
+const mapboxUrl = [
+    'https://api.mapbox.com/geocoding/v5/mapbox.places/',
+    '.json?access_token=pk.eyJ1Ijoicm9tYW5iYWNoYWxvcyIsImEiOiJja2xjNm82cnAwbWt6Mm9wMGZucWRlemdvIn0.YMTWniNZSYxZCYkLCijXiQ&limit=1'
+]
+
+// gets the coordinates by given address and calls a callback function
+const weatherForecast = (address, callback) => {
```

Comment:
1. Make this method pure (for ex., pass mapboxUrl as an argument)
2. Add validation for params you used. Is mapboxUrl[0] or mapboxUrl[1] really exists? Y
You can use lodash library for js and use methods like this
`_.has({ features: [{ center: [4,5,6] }, { center: [7,8,9] }] }, 'features[0].center[0]')`